### PR TITLE
Feature/sergeyro/chips hovered state + clickable

### PR DESCRIPTION
**Pull request by SergeyRoyt**
https://monday.monday.com/boards/3532714909/pulses/3591193843
Chips hover (+ update colors-description) + clickability

Need to check with care - potentially dangerous changes

### CODE REVIEW

File: src/utils/colors-vars-map.ts [hunk 1]
```diff
@@ -39,12 +40,14 @@ export const colorsMap = [
   },
   { color: "--positive-color-hover", description: "Use only as hover color on positive color" },
   { color: "--positive-color-selected", description: "Use only as selected indication for a positive colors" },
+  { color: "--positive-color-selected-hover", description: "description" }, // TODO
```
Comment (SergeyRoyt) [inline src/utils/colors-vars-map.ts:43]: TODO write description

File: src/components/Chips/Chips.tsx [hunk 2]
```diff
@@ -75,12 +76,21 @@ const Chips: VibeComponent<ChipsProps, HTMLElement> & {
...
-    const mergedRef = useMergeRefs({ refs: [ref, componentRef] });
+    const mergedRef = useMergeRefs({ refs: [ref, componentRef, hoverRef] });
 
     const backgroundColorStyle = useMemo(() => {
-      return { backgroundColor: disabled ? getCSSVar("disabled-background-color") : getElementColor(color, true) };
-    }, [disabled, color]);
+      let cssVar;
+      if (disabled) {
+        cssVar = getCSSVar("disabled-background-color");
+      } else if (isHovered) {
```
Comment (hadasfa) [inline src/components/Chips/Chips.tsx:87]: according to the design, hover state should be added only when the chip is clickable. This should happen only if there is a onMouseDown prop or onClick (which is not exist right now and we need to add)

File: src/components/Chips/Chips.tsx [hunk 3]
```diff
@@ -75,12 +76,21 @@ const Chips: VibeComponent<ChipsProps, HTMLElement> & {
...
+        cssVar = getElementColor(color, true, true);
```
Comment (hadasfa) [inline src/components/Chips/Chips.tsx:88]: not all colors having selected hover color. so I guess it will have a bad color for not supported colors. we need to add a check somehow if the color is not supported. I have an idea, let's talk about it in zoom?

File: src/components/Chips/Chips.tsx [hunk 4]
```diff
@@ -104,6 +120,7 @@ const Chips: VibeComponent<ChipsProps, HTMLElement> & {
         id={id}
         style={backgroundColorStyle}
         onMouseDown={onMouseDown}
+        onClick={onClick}
```
Comment (hadasfa) [inline src/components/Chips/Chips.tsx:123]: we know how to this accessible so let's not do it in a different pr (it's part of the definition of done)
You can add a clickable conditionally (just be aware that there is a different name to the data test id attribute0

File: src/components/Chips/Chips.tsx [hunk 5]
```diff
@@ -47,6 +46,18 @@ interface ChipsProps extends VibeComponentProps {
...
+   */
+  onClick?: (event: React.MouseEvent<HTMLDivElement, MouseEvent>) => void;
+  /**
+   * Applies when element has onClick or onMouseDown props
+   */
+  ariaLabel?: string;
+  /**
+   * Should element be focusable & clickable - for backward compatability
+   */
+  isClickable?: boolean;
```
Comment (SergeyRoyt) [inline src/components/Chips/Chips.tsx:60]: Added in order not to break anything old, which is using chip - e.g. Dropdown with multi-state
